components/Layout.js

```diff
@@ -71,20 +72,66 @@ export default class Layout extends React.Component {
             border: 3px solid #091f2f;
             font-family: 'Lora', serif;
             font-style: italic;
+            border-radius: 0px;
+          }
+
+          // geocoder style
+          .mapboxgl-ctrl-geocoder input {
+            background-image: url(https://www.boston.gov/crispus/images/public/icons/search.svg);
```

Comment:
Can you use `https://patterns.boston.gov/images/public/icons/search.svg` instead?

Reply:
👍